lib/permutations.rb

```diff
@@ -1,4 +1,7 @@
 
 def permutations?(string1, string2)
```

Comment:
This solution is O(n log n) which is good, but using a hash you can get it to O(n).  

I encourage you to think about how.